corrector.rb

```diff
@@ -0,0 +1,7 @@
+class Corrector
+  def correct_name(name)
+    name.strip! if name.size > 10
+
+    name.capitalize!
+  end
```

Comment:
- [ ] The method does not work as expected. It should return a capitalized string that has a maximum of 10 characters, but it only returns a capitalized string. Kindly fix this.

  ![yassine-corr](https://user-images.githubusercontent.com/36542986/156170834-905df2f3-ae6f-4225-ac92-8d21ad269d6f.jpg)